src/nodes.coffee

```diff
@@ -1906,12 +1928,19 @@ exports.RegexWithInterpolations = class RegexWithInterpolations extends Base
 
 exports.TaggedTemplateCall = class TaggedTemplateCall extends Call
   constructor: (variable, arg, soak) ->
-    arg = new StringWithInterpolations Block.wrap([ new Value arg ]) if arg instanceof StringLiteral
+    arg = StringWithInterpolations.fromStringLiteral arg if arg instanceof StringLiteral
```

Comment:
This is the existing `StringLiteral` -> `StringWithInterpolations` conversion (which didn't assign correct location data)

I think modeling this as a static method "pseudo-constructor" `StringWithInterpolations::fromStringLiteral()` seems appropriate

And `StringWithInterpolations::fromStringLiteral()` will see reuse when we cover JSX text content